aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -462,30 +472,32 @@ sub stop_and_remove_one_vms
             $main::this_app->info("Running VM will be removed: ",$t->name);
             $t->delete();
         } else {
-            $main::this_app->info("No QUATTOR flag found for Running VM: ",$t->name);
+            $main::this_app->info("No QUATTOR flag found for Running VM: ", $t->name);
```

Comment:
please add in the message that you stop processing and other `@runingvms` here?
but is the return really necessary?